src/custom/hooks/useIsActiveWallet.ts

```diff
@@ -9,7 +9,7 @@ export const useIsActiveWallet = (connection: Connection) => {
   if (!isActive) {
     return false
   } else if (isActive && !selectedWallet) {
-    return true
+    return false
```

Comment:
why this change?

Reply:
Because if I'm connected to WC wallet then I connect to Metamask, then I disconnect Metamask. It will show me that I'm connected to WC but I'm not really until I click on some WC wallet again.

<img width="1031" alt="Screenshot 2023-02-27 at 13 03 12" src="https://user-images.githubusercontent.com/34926005/221560515-267f473c-81cc-4ec5-957a-3598b881807b.png">

Reply:
Actually will update this